Formula/libslirp.rb

```diff
@@ -0,0 +1,38 @@
+class Libslirp < Formula
+  desc "General purpose TCP-IP emulator"
+  homepage "https://gitlab.freedesktop.org/slirp/libslirp"
+  url "https://elmarco.fedorapeople.org/libslirp-4.3.1.tar.xz"
+  sha256 "388b4b08a8cc0996cc5155cb027a097dc1a7f2cfe84b1121496608ab5366cc48"
+  license "BSD-3-Clause"
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "glib"
+
+  def install
+    system "sed", "-i", "-e", "s/,--version-script//g", "meson.build"
```

Comment:
Please use `inreplace` and try and patch this upstream as well.

Reply:
It was recently fixed upstream. I'll use `inreplace`.

Reply:
resolved.